### DISCUSSION TITLE
Cannot download files from site

### DISCUSSION THREAD

**Discussion by sztzco-01** [aclap-dev/video-downloadhelper#81] [Help and Q&A]
### Your question

Hi, I could download videos from this site on 2024 feb 11, but the latest update seems to fail. Could you please check? This link below is an example.
Thanks!

https://watch.magnvs.io/self-defense/videos/self-defense-trailer

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.0.20 (stable)
Browser locale: en-US
Production build
Built on 2024-02-12
Build options: browser=chrome, linuxlic=false, noyt=true, target=google

Found companion app: VdhCoApp 2.0.17
Companion app binary: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by paulrouget** [comment]
Thank you for reporting. I found the issue.

Please wait until the version 8.2.0.21 is released (in a few hours).

**Reply by sztzco-01** [reply]
Thanks!

**Comment by paulrouget** [comment]
See https://github.com/aclap-dev/video-downloadhelper/discussions/82
